### DISCUSSION TITLE
Pin code for settings section

### DISCUSSION THREAD

**Discussion by Hellfish78** [syssi/esphome-jk-bms#164] [General]
Hi, first of all thank you very much for this fantastic software! In the original jk bms app i can protect the settings with a pin code which makes absolute sense because you can crash your expensive battery with the wrong settings. Is it possible to implement a pin code in this config too? Best regards, the flying hellfish

**Comment by syssi** [comment]
Are you aware of this issue #139? The BMS doesn't care about pin numbers. There is no authorization required to change the BMS settings.

**Reply by Hellfish78** [reply]
Oh ok, i did know that the ble connection does not need any authentication but that the settings even has no protection is new to my. Thanks for the info! So i guess there is no way to protect the settings because it is a design fail in the bms?

**Reply by syssi** [reply]
IMO yes.
